deploy/Dockerfile

```diff
@@ -0,0 +1,112 @@
+# syntax=docker/dockerfile:1
+
+ARG BASE_IMAGE_TAG
+
+ARG OIE_RELEASE_VERSION
+
+ARG UID=14285
+ARG GID=14285
```

Comment:
Expected behavior is that UID and GID are 1000 so that bind mount permissions work for the majority of users where their host UID is also 1000.

See https://github.com/nextgenhealthcare/connect-docker/issues/49. This caused quite a few issues in the NextGen slack when the ubuntu image started including an `ubuntu` user with UID 1000 in their base images, so NextGen bumped the mirth UID to 1001.

Here is where they corrected it by renaming the `ubuntu` user to `mirth` so that the mirth user retained UID 1000. https://github.com/nextgenhealthcare/connect-docker/commit/233502c85794c7485d3b48120d97f38fa8fc177e

Reply:
the other piece of that is even if you aren't using bind mounts, and you try to use existing volumes from a nextgen container, but the nextgen container had a different UID from the new oie container, the oie user won't have appropriate access to the existing volume.